packages/font-support-info/src/index.ts

```diff
@@ -0,0 +1,1090 @@
+export interface IFontInfoData {
+  name: string;
+  support_languages: string[];
+}
+
+const FONTS_INFO_LIST = {
+  abhaya_libreregular: {
+    name: 'Abhaya Libreregular',
```

Comment:
Why is this `abhaya_libreregular`? Is there a bold variant of it? If not we can name it `abhaya_libre`

Reply:
This is the font family name we are using in creately-next. we have to change the name in all the places if we change the name here

Reply:
Noted. We can change the name to `Abhaya Libre Regular` then